src/plugins/embeddable/public/lib/attribute_service/attribute_service.tsx

```diff
@@ -29,21 +29,37 @@ import {
  */
 export const ATTRIBUTE_SERVICE_KEY = 'attributes';
 
-export interface AttributeServiceOptions<A extends { title: string }> {
+export interface GenericAttributes {
+  title: string;
+}
+export interface AttributeServiceUnwrapResult<
+  SavedObjectAttributes extends GenericAttributes,
+  MetaInfo extends unknown = unknown
+> {
+  attributes: SavedObjectAttributes;
+  metaInfo?: MetaInfo;
+}
+export interface AttributeServiceOptions<
+  SavedObjectAttributes extends { title: string },
```

Comment:
Did you mean to? 

```suggestion
  SavedObjectAttributes extends GenericAttributes,
```

Reply:
Ah yes, looks good, I've committed this change!